.github/pr-labeler.yml

```diff
@@ -0,0 +1,90 @@
+# archived/webui:
+# backport/1.6:
+# backport/1.7:
+# backport/1.8:
+# beta/1.8:
+# blocks-release:
+# crash:
+# do-not-merge:
+# help-wanted:
+# needs-discussion:
+# needs-investigation:
+# post-beta:
+pr/dependencies:
+    - vendor/**/*

```

Comment:
Also should include `go.mod` and `go.sum` changes in the root and also the submodules.

Reply:
Should we remove/combine with `theme/go-modules` https://github.com/hashicorp/consul/pull/8709/files#diff-2231aec93767d94c46596156d32d0792R43